src/napari/layers/base/base.py

```diff
@@ -1431,6 +1433,20 @@ def _slice_dims(
                 extent=True,
             )
 
+    def _world_to_layer_dims(
+        self,
+        *,
+        world_dims: Sequence[int] | Array1dOfInts,
+        ndim_world: int,
+    ) -> Array1dOfInts:
+        """For docs to build
+
+        Remove after clean guides/rendering notebook
+        """
```

Comment:
Not sure what "clean guides/rendering notebook" means, can you add some extra detail?

Reply:
This function is used in guides/rendering notebook https://github.com/napari/docs/blob/main/docs/guides/rendering.md